README.uk-UA.md

```diff
@@ -0,0 +1,211 @@
+            url "https://cardinalcommerceprod.jfrog.io/artifactory/android"
+            credentials {
+                username 'braintree_team_sdk'
+                password 'AKCp8jQcoDy2hxSWhDAUQKXLDPDx6NYRkqrgFLRc3qDrayg6rrCbJpsKKyMwaykVL8FWusJpp'
+            }
+        }
+    }
+}
+```
+
+У вашому `AndroidManifest.xml`, `android:allowBackup="false"` можна замінити `android:allowBackup="true"`, він відповідає за резервне копіювання програми.
+
+Крім того, додайте цей фільтр намірів до основної діяльності в `AndroidManifest.xml`
+
+```xml
+<activity>
+    ...
+    <intent-filter>
+        <action android:name="android.intent.action.VIEW" />
+        <category android:name="android.intent.category.DEFAULT" />
+        <category android:name="android.intent.category.BROWSABLE" />
+        <data android:scheme="${applicationId}.braintree" />
+    </intent-filter>
+</activity>
+
+```
+**ПРИМІТКА: Платежі карткою не працюють на пристроях з рутованим доступом і емуляторах Android**
+
+Якщо ваш проект використовує Progurad, додайте наступні рядки у `proguard-rules.pro` файл
+```
+-keep class com.cardinalcommerce.dependencies.internal.bouncycastle.**
+-keep class com.cardinalcommerce.dependencies.internal.nimbusds.**
+-keep class com.cardinalcommerce.shared.**
+```
+
+## Особливо для iOS
+```bash
+cd ios
+pod install
+```
+###### Налаштуйте нову схему URL-адреси
+Додайте схему URL-адреси пакета {BUNDLE_IDENTIFIER}.payments у свою програму Info через XCode або вручну в Info.plist. У вашому Info.plist ви повинні мати щось на зразок:
+
+```xml 
+<key>CFBundleURLTypes</key>
+<array>
+    <dict>
+        <key>CFBundleTypeRole</key>
+        <string>Editor</string>
+        <key>CFBundleURLName</key>
+        <string>com.myapp</string>
+        <key>CFBundleURLSchemes</key>
+        <array>
+            <string>com.myapp.payments</string>
+        </array>
+    </dict>
+</array>
+```
+###### Оновіть свій код
+У вашому `AppDelegate.m`:
+
+```objective-c
+#import "BraintreeCore.h"
+
+...
+- (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions
+{
+    ...
+    [BTAppContextSwitcher setReturnURLScheme:self.paymentsURLScheme];
+}
+
+- (BOOL)application:(UIApplication *)application
+            openURL:(NSURL *)url
+            options:(NSDictionary<UIApplicationOpenURLOptionsKey,id> *)options {
+
+    if ([url.scheme localizedCaseInsensitiveCompare:self.paymentsURLScheme] == NSOrderedSame) {
+        return [BTAppContextSwitcher handleOpenURL:url];
+    }
+    
+    return [RCTLinkingManager application:application openURL:url options:options];
+}
+
+- (NSString *)paymentsURLScheme {
+    NSString *bundleIdentifier = [[NSBundle mainBundle] bundleIdentifier];
+    return [NSString stringWithFormat:@"%@.%@", bundleIdentifier, @"payments"];
+}
+```
+
+
+## Використання
+
+##### Показати модуль PayPall
```

Comment:
@vasylnahuliak I guess it should be the **PayPal** also this typo is present in original README

Reply:
PayPal is correct. Thanks!